viridian/1.2.2/README.md

```diff
@@ -0,0 +1,47 @@
+# Viridian container
+
+Main tool: [Viridian](https://github.com/iqbal-lab-org/viridian)
+  
+Code repository: [Viridian](https://github.com/iqbal-lab-org/viridian)
+
+Additional tools:
+- samtools: version
+- bcftools: version
+- htslib: version
+- ENA: 1.7.1
+- ngmerge: version
+- vt: 0.577721
+- racon: 1.5.0
+- mummer: 4.0.0
+- read-it-and-keep: 
+- cylon: commit hash 57d559a76254b0b95785f7c02fa58ef806713e01
+- varifier: commit hash 8bc8726ed3cdb337dc47b62515e709759e451137
+- minimap2: version
```

Comment:
I've updated the README to match the template a bit better. We try to keep track of additional tools that get explicitly installed in an image.

In the Dockerfile that I pasted into chat, I used a different samtools, bcftools, htslib, ngmerge, read-it-and-keep, and minimap2, so these are general here until a final version is settled on.

This will need to be updated before merged.